### DISCUSSION TITLE
Strange behavior of LEDs.

### DISCUSSION THREAD

**Discussion by c7v** [arendst/Tasmota#17501] [Q&A] [answered]
Hi!

I have a board ESP82266 on which the firmware [12.3.1 tasmota-RU.bin](https://github.com/arendst/Tasmota/releases/tag/v12.3.1), LED strip name WS2812.

1. When I turn on yellow, red turns on.
2. But the strange behavior of the LED strip, when turned on, the light flickers, it feels like someone is pulling the brightness.

The main question in problem number 2, why is this happening?

**Comment by sfromis** [comment, marked as answer]
WS2812 strips have no yellow, that is formed via red+green. The resulting "yellow" color is often not great, but you may be able to improve it by changing the balance of red and green.

Not clear to me what your point 2 operationally means, but it could indicate an issue with the need for having enough power with the right voltage and good connections.

**Reply by c7v** [reply]
The tape is powered from the board pin VIN and GND. I used to have my firmware and there were no problems with the yellow light and voltage. I also found that the area marked on the board with a circle heats up to +120 degrees Celsius.

![nodemcu-v3-cp2102-1-500x500](https://user-images.githubusercontent.com/50216354/209475964-b7d73b32-8c63-4a4f-8721-0825e396163f.png)

**Reply by sfromis** [reply]
Unless only a low number of pixels, do not draw power from VIN, that could cause the board to be damaged. Absolute max 1A

**Reply by c7v** [reply]
I agree, since there are 90 LEDs on one tape, only 30 on the second. Additionally, as it turned out, I had a bad connection with the contacts. Fixed everything.

Thanks for the help!
